app.py

```diff
@@ -20,87 +16,66 @@ def on_run_app():
         beers = {}
 
 
-def on_exit_app():
+@app.on_event("shutdown")
+def shutdown_event():
     with open("data.json", "w+") as outfile:
         json.dump(beers, outfile)
 
 
-atexit.register(on_exit_app)
+@app.get("/beers/{isbn}")
+async def get_beer(isbn: str):
+    if isbn in beers:
+        return beers[isbn]
+    raise HTTPException(status_code=404, detail="Item not found")
 
 
-@app.route('/beers')
-def list_beer():
+@app.get("/beers")
+async def list_beer(req: Request):
```

Comment:
It's list of beers (plural)
```suggestion
async def list_beers(req: Request):
```